<launch>
  <arg name="camera" default="zed"/>
  <arg name="tf_prefix" default="" />
  <arg name="pi/2" value="1.5707963267948966" />
  <arg name="optical_rotate" value="0 0 0 -$(arg pi/2) 0 -$(arg pi/2)" />

  <node pkg="tf" type="static_transform_publisher" name="$(arg camera)_base_link1"
    args="0 0 0 0 0 0 1 camera_link $(arg tf_prefix)/$(arg camera)_frame 100" />

  <node pkg="tf" type="static_transform_publisher" name="$(arg camera)_base_link2"
    args="$(arg optical_rotate) $(arg tf_prefix)/$(arg camera)_frame $(arg tf_prefix)/$(arg camera)_optical_frame  100" />

  <arg name="computeDepth" value="1"/>
  <arg name="svo_file" default=""/>
  
  <!--<group ns="camera">
    <node name="zed_wrapper_node" pkg="zed_wrapper" type="zed_wrapper_node" args="$(arg computeDepth) $(arg svo_file)" output="screen">

      <param name="resolution"            value="2" />
      <param name="quality"               value="2" />
      <param name="sensing_mode"          value="1" />
      <param name="frame_rate"            value="15" />

      <param name="left_topic"            value="rgb/image_rect_color" />
      <param name="second_topic"          value="depth_registered/image_raw" />
      <param name="left_cam_info_topic"   value="rgb/camera_info" />
      <param name="second_cam_info_topic" value="depth/camera_info" />
      <param name="left_frame_id"         value="/zed_optical_frame" />
      <param name="second_frame_id"       value="/zed_optical_frame" />

    </node>
  </group>-->
</launch>
